acquia/acquia_cms
# ACMS-1108: Remove config moved to acquia_cms modules

**Motivation**
Fixes #ACMS-1108

**Proposed changes**
Remove config moved to acquia_cms_* module from acquia_cms profile 

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch and latest acquia_cms_* modules
- verify everything works well

**Merge requirements**
